### ISSUE TITLE
Integration Tests Failing from Build Workflow

### ISSUE THREAD

**Issue by 0Hughman0** [jupyterlab/extension-template#40]
## Description

The build workflow fails with my JupyterLab extension on the integration tests job.

I think you can see a workflow runs here?:

https://github.com/0Hughman0/jupyter_cassini/actions/runs/5823678174/job/15791243849

I get both

```
[WebServer] [I 2023-08-10 16:35:28.678 ServerApp] Registered HelloWorld extension at URL path /jupyter_cassini_server
[I 2023-08-10 16:35:28.678 ServerApp] jupyter_cassini_server | extension was successfully loaded.
```

In the log, indicating to me that extensions are activating correctly.

However, even the [extension activates](https://github.com/0Hughman0/jupyter_cassini/blob/00a7b22898d58a2a61dcf56c5b2ffb91ac936f1d/ui-tests/tests/jupyter_cassini_server.spec.ts#L9) test fails.

Looking at the videos generated by the Playwright output, it appears like the extension doesn't activate. 

## Expected behavior

I would expect these tests to pass as they pass on my local system (is Windows 10?)

### Context

One potential cause, that I haven't been able to diagnose is for my extension to run, I have to modify the `jlpm start` command:

https://github.com/0Hughman0/jupyter_cassini/blob/00a7b22898d58a2a61dcf56c5b2ffb91ac936f1d/ui-tests/package.json#L7

Which launches the server and configures it to work with Cassini, which is the backend for this project. But I haven't made any changes to the `build.yml` workflow and the tests run fine on my local machine.

Thanks for your help, or any suggestions on how to diagnose.

Hugh

**Comment by fcollonval** [comment]
Hey @0Hughman0 this bug is due to the release of JupyterLab 4.0.4. It has been fixed in 4.0.5 but you will need to upgrade `@jupyterlab/galata` to version 5.0.5 to fix it. You can achieve this by executing `jlpm up "@jupyterlab/galata"` in your integration tests folder.

**Comment by 0Hughman0** [comment]
Thanks!

Yes they seem to be running now... still failing... but I think this time at least it's because of something I've done!

Thanks for the help.
